### DISCUSSION TITLE
can't install strongswan with tar.gz format

### DISCUSSION THREAD

**Discussion by sokolblyat9** [strongswan/strongswan#1753] [General]
after unpacking the file, entering a banal command from the documentation - ./configure --prefix=/usr --sysconfdir=/etc and getting the make command, the following error appears:
gcc: fatal error: cannot specify '-o' with '-c', '-S' or '-E' with multiple files
compilation aborted.
Tell me how to fix it?
The full error looks like this:
ilya@pcashnik:~/Desktop/strongswan-5.9.11$ make
make everything recursive
make[1]: enter directory "/home/ilya/Desktop/strongswan-5.9.11"
Doing everything in src
make[2]: enter directory "/home/ilya/Desktop/strongswan-5.9.11/src"
We do everything in .
make[3]: enter directory "/home/ilya/Desktop/strongswan-5.9.11/src"
make[3]: The "all-am" target does not require a command to be executed.
make[3]: exit directory "/home/ilya/Desktop/strongswan-5.9.11/src"
All inclusive
make[3]: enter directory "/home/ilya/Desktop/strongswan-5.9.11/src/include"
make[3]: The "all" target does not require a command to be executed.
make[3]: exit directory "/home/ilya/Desktop/strongswan-5.9.11/src/include"
Doing everything in libstrongswan
make[3]: enter directory "/home/ilya/Desktop/strongswan-5.9.11/src/libstrongswan"
make everything recursive
make[4]: enter directory "/home/ilya/Desktop/strongswan-5.9.11/src/libstrongswan"
We do everything in .
make[5]: enter directory "/home/ilya/Desktop/strongswan-5.9.11/src/libstrongswan"
depbase=echo library.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||';
/bin/bash ../../libtool --tag=CC --mode=compile gcc -DHAVE_CONFIG_H -I. -I../.. -I../../src/libstrongswan -DIPSEC_DIR="/usr/libexec/ipsec" -DIPSEC_LIB_DIR="/usr/lib/ipsec" -DPLUGINDIR="/usr/lib/ipsec /plugins" -DSTRONGSWAN_CONF="/etc/strongswan.conf" -Wno-format -Wno-format-security -Wno-implicit-fallthrough -Wno-missing-field-initializers -Wno-pointer -sign -Wno-sign-compare -Wno-type-limits -Wno-unused-parameter -g -O2 -include /home/ilya/Desktop/strongswan-5.9.11/config.h -MT library.lo -MD -MP -MF $depbase.Tpo -c -o library.lo library.c &&
mv -f $depbase.Tpo $depbase.Plo
libtool: compile: gcc -DHAVE_CONFIG_H -I. -I../.. -I../../src/libstrongswan -DIPSEC_DIR="/usr/libexec/ipsec" -DIPSEC_LIB_DIR="/usr/lib/ipsec" -DPLUGINDIR="/usr/lib/ipsec /plugins" -DSTRONGSWAN_CONF="/etc/strongswan.conf" -Wno-format -Wno-format-security -Wno-implicit-fallthrough -Wno-missing-field-initializers -Wno-pointer -sign -Wno-sign-compare -Wno-type-limits -Wno-unused-parameter -g -O2 -include /home/ilya/Desktop/strongswan-5.9.11/config.h -MT library.lo -MD -MP -MF .deps/library .Tpo -c library.c -fPIC -DPIC -o .libs/library.o
gcc: fatal error: cannot specify '-o' with '-c', '-S' or '-E' with multiple files
compilation aborted.
make[5]: *** [Makefile:2184: library.lo] Error 1
make[5]: exit directory "/home/ilya/Desktop/strongswan-5.9.11/src/libstrongswan"
make[4]: *** [Makefile:2273: all-recursive] Error 1
make[4]: exit directory "/home/ilya/Desktop/strongswan-5.9.11/src/libstrongswan"
make[3]: *** [Makefile:1353: all] Error 2
make[3]: exit directory "/home/ilya/Desktop/strongswan-5.9.11/src/libstrongswan"
make[2]: *** [Makefile:539: all-recursive] Error 1
make[2]: exit directory "/home/ilya/Desktop/strongswan-5.9.11/src"
make[1]: *** [Makefile:604: all-recursive] Error 1
make[1]: exit directory "/home/ilya/Desktop/strongswan-5.9.11"
make: *** [Makefile:515: all] Error 2

**Comment by tobiasbrunner** [comment]
No idea. But what version of GCC and perhaps libtool and make do you use?
